Makefile

```diff
@@ -5,12 +5,6 @@ else
 	FILTERARG =
 endif
 
-ifdef DOCSVERSION
```

Comment:
I think we used to have a `make docs` target in here where this was used, but it's irrelevant now